.github/workflows/linux.yml

```diff
@@ -4,28 +4,35 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         cxx: [g++-4.8, g++-8, g++-10, clang++-9]
         build_type: [Debug, Release]
         std: [11]
+        os: [ubuntu-18.04]
         include:
           - cxx: g++-4.8
             install: sudo apt install g++-4.8
+            os: ubuntu-18.04
```

Comment:
Can we avoid explicitly specifying `ubuntu-18.04` here and below? Won't it get the default from `os` above?

Reply:
My tests with Actions show that no, it won't be defaulted to `ubuntu-18.04`, it's just empty in those cases.
[Here is the CI run](https://github.com/alexezeder/fmt/actions/runs/502526196) for [config without explicit `os`](https://github.com/alexezeder/fmt/commit/792456479b0e52be84a1c4face507c29bfd2c7d3).